templates/firefox-ios.xliff

```diff
@@ -4044,14 +4106,34 @@ Private Tab</source>
         <source>Learn more about Firefox Suggest</source>
         <note>In the search page of the Settings menu, the title for the link to the SUMO Page about Firefox Suggest.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.SearchBrowsingHistory.Title.v124" xml:space="preserve">
+        <source>Search Browsing History</source>
+        <note>In the Search page of the Settings menu, the title for the setting to enable search browsing history.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.SearchSearchBookmarks.Title.v124" xml:space="preserve">
+        <source>Search Bookmarks</source>
+        <note>In the Search page of the Settings menu, the title for the setting to enable search bookmarks.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.SearchSyncedTabs.Title.v124" xml:space="preserve">
+        <source>Search Synced Tabs</source>
+        <note>In the Search page of the Settings menu, the title for the setting to enable synced tabs.</note>
+      </trans-unit>
       <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Description.v124" xml:space="preserve">
         <source>Get suggestions from the web related to your search</source>
         <note>In the Search page of the Settings menu, the description for the setting to enable Suggestions from Firefox.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Description.v124.v2" xml:space="preserve">
+        <source>Get suggestions from %@ related to your search</source>
+        <note>In the Search page of the Settings menu, the description for the setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.</note>
+      </trans-unit>
       <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124" xml:space="preserve">
         <source>Suggestions from %@</source>
         <note>In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124.v2" xml:space="preserve">
+        <source>Suggestions from the Web</source>
+        <note>In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox.</note>
```

Comment:
Maybe nit: should the comment rather say "(...)the title for setting to enable Suggestions from the Web" vs Firefox?

Reply:
Addressed! If you extract again, the update will appear 🪄  :D